escrow/project/escrow-contract/tests/functions/propose_arbiter.rs

```diff
@@ -475,11 +475,11 @@ mod revert {
         )
         .await;
         let asset = create_asset(defaults.asset_amount, defaults.asset_id).await;
-        let tx_params = TxParameters::new(None, Some(1_000_000), None);
+        let tx_params = TxParameters::new(None, Some(10_000_000), None);
```

Comment:
The gas usages have changed between the versions and some tests were failing because of insufficient gas or gas limit being too low. We never provide exact gas values in tests, so I simply increased the lower limit for transactions parameters and increased forwarded gas where needed. I can provide more specific values if needed. What do you think?